biojava-structure/src/main/java/org/biojava/nbio/structure/io/LocalPDBDirectory.java

```diff
@@ -687,6 +690,14 @@ public File getLocalFile(String pdbId) {
 				for(String ex : getExtensions() ){
 					File f = new File(searchdir,prefix + pdbId.toLowerCase() + ex) ;
 					if ( f.exists()) {
+						// delete files that are too short to have contents
+						if( f.length() < MIN_PDB_FILE_SIZE ) {
+							boolean success = f.delete();
+							if( ! success) {
+								return null;
```

Comment:
Shouldn't this rather throw an exception (perhaps IOException)? I don't see how can this null be handled if file can't be deleted.

Reply:
switched to `Files.delete` which throws the exception